cmake/external/date.cmake

```diff
@@ -0,0 +1,16 @@
+include(ExternalProject)
+set(DATE_PREFIX ${CMAKE_BINARY_DIR}/date)
+set(DATE_SOURCE_DIR "${THIRD_PARTY_DIR}/date")
+set(DATE_INCLUDE_DIR "${DATE_SOURCE_DIR}/include")
+ExternalProject_Add(
+        date
+        PREFIX ${DATE_PREFIX}
+        SOURCE_DIR ${DATE_SOURCE_DIR}
+        GIT_REPOSITORY https://github.com/HowardHinnant/date.git
```

Comment:
seems ok to me to have this

Reply:
hahahaha
Ah man, finally in c++20, c++ will have nice date parsing... things are changing fast these days ;)